packages/apollos-data-connector-postgres/src/features/migrations/005_update_features_unique_index.js

```diff
@@ -0,0 +1,21 @@
+async function up({ context: queryInterface }) {
+  await queryInterface.removeIndex('feature', 'features_parent_id_type_data');
+  await queryInterface.addIndex(
+    'feature',
+    ['parent_id', 'type', 'data', 'priority'],
```

Comment:
What do you think about moving `priority` before `data` and `type`? That way it can be used for sorting when filtering by `parent_id`
```suggestion
    ['parent_id', 'priority', 'type', 'data'],
```

Reply:
Yeah sure. I didn't know that it worked that way but that sounds good.

Reply:
This might help explain the effect/impact https://blog.timescale.com/blog/use-composite-indexes-to-speed-up-time-series-queries-sql-8ca2df6b3aaa/